api/tests/core/external/external_users_test.py

```diff
@@ -605,3 +611,20 @@ def test_get_most_favorite_subcategories_two_equal():
         subcategories.SEANCE_CINE.id,
         subcategories.FESTIVAL_MUSIQUE.id,
     }
+
+
+def test_get_user_attributes_with_achievements(db_session):
+    user = BeneficiaryFactory()
+    booking = BookingFactory(user=user)
+    AchievementFactory(
+        user=user,
+        booking=booking,
+        name=AchievementEnum.FIRST_MOVIE_BOOKING,
+        unlockedDate=datetime(2025, 1, 1, 12, 0, 0),
+        seenDate=datetime(2025, 1, 2, 12, 0, 0),
+    )
+
+    attributes = get_user_attributes(user)
+    assert isinstance(attributes.achievements, list)
+    assert len(attributes.achievements) == 1
+    assert attributes.achievements[0] == "FIRST_MOVIE_BOOKING"
```

Comment:
Ces 3 lignes se résument en : 
```suggestion
    assert attributes.achievements == ["FIRST_MOVIE_BOOKING"]
```